website/project/model.py

```diff
@@ -84,36 +83,32 @@ class MetaSchema(StoredObject):
     schema = fields.DictionaryField()
     category = fields.StringField()
 
-    # Version of the Knockout metadata renderer to use (e.g. if data binds
-    # change)
+    # Deprecated legacy field
```

Comment:
If this field is deprecated, what is the advantage of keeping it as part of the model?